src/shared/components/MultiFileSelect/MultiFileSelect.tsx

```diff
@@ -189,14 +198,14 @@ export const MultiFileSelect: React.FC<MultiFileSelectProps> = ({
         maxSize={step === 'video' ? maxVideoSize : maxImageSize}
         onUploadFile={handleUploadFile}
         onReAdjustThumbnail={handleReAdjustThumbnail}
-        isLoading={isLoading}
+        isLoading={isVideoLoading || isImgLoading}
         fileType={step}
         title={step === 'video' ? 'Select video file' : 'Add thumbnail image'}
         thumbnailUrl={files.thumbnail?.url}
         paragraph={
           step === 'video'
-            ? `Maximum 10GB. Accepts any format supported by your browser.`
-            : `Accepts any format supported by your browser.`
+            ? `Maximum 10GB. Prefered format is MP4 (H.264) or WEBM (VP8,VP9)`
```

Comment:
Let's switch that around and suggest webm first:
```suggestion
            ? `Maximum 10GB. Preferred format is WebM (VP9/VP8) or MP4 (H.264)`
```